app/models/invenio_rdm_collection_converter.rb

```diff
@@ -0,0 +1,18 @@
+include Sufia::Export
+
+# Convert a Collection including metadata, permissions and version metadata into a PORO
+# so that the metadata can be exported in json format using to_json
+#
+class InvenioRdmCollectionConverter < Sufia::Export::Converter
+  # Create an instance of a InvenioRdmCollectionConverter converter containing all the metadata for json export
+  #
+  # @param [collection] collection file to be converted for export
+  def initialize(collection)
```

Comment:
Just curious, are the variables in the 'initialize' function commented out because you're testing?

Reply:
I'm leaving the commented out variables so we have an easy reference to the fields that are available from ActiveFedora  on the Collection and GenericFile objects. For me personally I like to have it in the code so I don't have to open a console or a webpage.